.buildkite/pipelines/flaky_tests/pipeline.ts

```diff
@@ -55,6 +68,7 @@ function getTestSuitesFromJson(json: string) {
   const testSuites: Array<
     | { type: 'group'; key: string; count: number }
     | { type: 'ftrConfig'; ftrConfig: string; count: number }
+    | { type: 'scoutConfig'; scoutConfig: string; count: number }
```

Comment:
When Scout config is selected in Flaky test runner, server should return in json:

```
{
  type: 'scoutConfig',
  title: 'Scout: path/to/config.ts',
  id: 'scout:path/to/config.ts',
  scoutConfig: 'path/to/config.ts',
  count: 15
}
```